client/components/business-profile.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default function BusinessProfile(props) {
+  return (
+    <>
+      <div className="row">
+        <div className="column-full">
+          <div className="graphic">
+            <div className="rname">
+              <h1 className="h1-title">WaitingFuze</h1>
+            </div>
+          </div>
```

Comment:
I assume there should be some methods in here so that the user can submit the form. Maybe it just hasn't been committed yet?